src/api/article/controller.js

```diff
@@ -0,0 +1,57 @@
+import { checkIsFound } from 'utils/validation';
+import { sendJson } from 'utils/api';
+import Article, { serializeArticle } from './article.model';
+
+export const getAll = (req, res, next) => {
+  const page = parseInt(req.query.page, 10) || 0;
+  const pageSize = parseInt(req.query.pageSize, 10) || 10;
+  const skip = page * pageSize;
+  let result;
+
+  return Article
+    .find({})
+    .populate('type')
+    .skip(skip)
+    .limit(pageSize)
+    .then(articles => articles.map(serializeArticle))
```

Comment:
I try this, but there are several problems
1. duplication, we can reuse it only as `export string` with `Article` fields, WTF??
2. select doesn't work with `populate` (for me)